pkg/server/commands/list_objects.go

```diff
@@ -315,19 +317,22 @@ func (q *ListObjectsQuery) internalCheck(
 	objectsFound *uint32,
 	resultsChan chan<- string,
 ) error {
-	query := NewCheckQuery(q.Datastore, q.Meter, q.Logger, q.ResolveNodeLimit)
 
-	resp, err := query.Execute(ctx, &openfgapb.CheckRequest{
-		StoreId:              req.GetStoreId(),
-		AuthorizationModelId: req.GetAuthorizationModelId(),
+	resp, err := q.CheckResolver.ResolveCheck(ctx, &graph.ResolveCheckRequest{
+		StoreID:              req.GetStoreId(),
+		AuthorizationModelID: req.GetAuthorizationModelId(),
 		TupleKey:             tuple.NewTupleKey(tuple.ObjectKey(obj), req.GetRelation(), req.GetUser()),
-		ContextualTuples:     req.GetContextualTuples(),
+		ContextualTuples:     req.GetContextualTuples().GetTupleKeys(),
+		ResolutionMetadata: &graph.ResolutionMetadata{
+			Depth: q.ResolveNodeLimit,
+		},
 	})
 	if err != nil {
 		// ignore the error. we don't want to abort everything if one of the checks failed.
 		q.Logger.ErrorWithContext(ctx, "check_error", zap.Error(err))
```

Comment:
Do we want to log these as errors?

Reply:
why not? 🤔

Reply:
We should at least fix the error message. The error message should simply be the error 😄 . I shouldn't have to look at the `error` field of the logged message.

Reply:
Now I am not sure. Writing aloud for a sec: if a check fails and it was `allowed: true` then ignoring it may mean we return a result which does not contain all the results. This could be bad if one wants to use this result to delete objects a particular user has access to or something...

Reply:
Now we have https://github.com/orgs/openfga/discussions/110, but obviously this is not the PR to solve this question.

Reply:
Will trace this work as part of that discussion 👍